src/holoscan_cli/common/artifact_sources.py

```diff
@@ -42,10 +39,10 @@ class ArtifactSources:
 
     def __init__(self) -> None:
         self._logger = logging.getLogger("common")
-        self._supported_holoscan_versions = ["2.6.0", "2.7.0", "2.8.0"]
+        self._supported_holoscan_versions = ["3.0.0"]
```

Comment:
Holoscan SDK 2.8.0 is the latest public release, will this make the `holoscan-cli` package on PyPI unusable until 3.0 is released?

Reply:
I have updated this to 2.9 in another PR.